kitsune/wiki/tests/test_templates.py

```diff
@@ -1013,6 +1013,143 @@ def test_keywords_require_permission(self):
         new_rev = Revision.objects.filter(document=doc).order_by('-id')[0]
         eq_('keyword1 keyword2', new_rev.keywords)
 
+    def test_restore_draft_revision(self):
+        rev = ApprovedRevisionFactory(is_ready_for_localization=True)
+        doc = rev.document
+        doc.is_localizable = True
+        doc.save()
+        # Check the Translation page has Save Draft Button
+        trans_url = reverse('wiki.translate', locale='bn-BD', args=[doc.slug])
+        trans_resp = self.client.get(trans_url)
+        eq_(200, trans_resp.status_code)
+        trans_content = pq(trans_resp.content)
+        eq_(0, len(trans_content('.user-messages li')))
+        eq_(1, len(trans_content('.submit .btn-draft')))
+        data = {
+            'content': 'Test Content bla bla bla',
+            'keywords': 'something',
+            'summary': 'some summary',
+            'title': 'test title',
+            'slug': 'test-slug',
+            'based_on': int(rev.id)}
```

Comment:
I strongly believed it would be int. but sometime the test fail because it return ``45L`` or something like that.